src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball { "
+                + " color = "
+                + color
+                + ", number = "
+                + number
+                + " }";
+    }
+
+    public enum Color {
+        RED, BLUE, GREEN, YELLOW, ORANGE, PURPLE, PINK;
+    }
```

Comment:
According to the Java style guide, enum values should be written in uppercase letters. The `Color` enum constants are correctly named here, adhering to the style guide.